projects/collection-editor-library/src/lib/components/meta-form/meta-form.component.ts

```diff
@@ -334,6 +318,35 @@ export class MetaFormComponent implements OnChanges, OnDestroy {
     return response;
   }
 
+  valueChanges(event: any) {
+    if (_.has(event, 'shuffle')) {
+      this.showShuffleMessage(event);
+    }
+    const data = _.omit(event, ['allowECM', 'levels', 'setPeriod']);
+    if (!_.isEmpty(event?.levels)) {
+      data.outcomeDeclaration = {
+        levels: this.createLeavels(event.levels)
+      };
+    }
+    event?.instance ? data.instances = { label : event?.instances } : '';
```

Comment:
why are we assigning empty string to `data.instances` ?

Reply:
if the creator entered the instance name it will be set else it will be empty string